nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/nf-common.js

```diff
@@ -102,7 +102,6 @@
                 type: 'DELETE',
                 url: '../nifi-api/access/logout',
             }).done(function () {
-                nfStorage.removeItem("jwt");
```

Comment:
Is there a reason for not the JWT token after a successful logout?  It seems like that should still occur.

Reply:
Ah yes, I'll add that back. I had removed the local storage JWT temporarily.